frontend/src/e2e-test/pages/employee/placement-draft-page.ts

```diff
@@ -13,6 +14,13 @@ export class PlacementDraftPage {
     '[data-qa="restricted-details-warning"]'
   )
 
+  setStartDate = async (date: LocalDate) => {
```

Comment:
Unnecessary wrapper, expose a public field instead: `readonly datePicker = this.page.findByDataQa('start-date')`

Reply:
Fixed